src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -112,8 +115,10 @@ class CrashHeader extends React.Component {
 
     return (
       <div className="crash-title">
-        {beforeTitle}
-        {titleNode}
+        <StyledTitleInfo>
```

Comment:
This could just be `TitleInfo` - personally I add `Styled` when I'm trying to extend another styled component and can't immediately think of a better name.

Reply:
good point 🙂